js/new-register.js

```diff
@@ -204,15 +204,16 @@ $("#btnSubmit").click(function(){
     request.done(function (response, textStatus, jqXHR){
         // Log a message to the console
         console.log(response);
-        if(response == 1){
+        if("success" in response){
             //user is logged in
 			//redirect to account page
+            window.localStorage.setItem('clubName',$("#inputClubName").val());
```

Comment:
Why???
It is not necessary that the club who registers, will be the one who logs in